### DISCUSSION TITLE
How and where to record project/milestone objectives

### DISCUSSION THREAD

**Discussion by rsjethani** [GothenburgBitFactory/taskwarrior#3616] [Q&A]
Hello,

Whenever I start a new project, I like to record objectives, initial thoughts, challenges etc. and only then carve out tasks, timeline etc.
As a newbie I see that I can use project tag to collect tasks related to a project but then where and how to record above information?

Thank you.

**Comment by thehunmonkgroup** [comment]
I consider the lack of a decent 'notes' feature to be the biggest missing piece in Taskwarrior. 

I wrote https://github.com/thehunmonkgroup/onenote to address that deficiency for my own needs.
